lnrpc/invoicesrpc/invoices.pb.go

```diff
@@ -37,7 +37,7 @@ func (m *CancelInvoiceMsg) Reset()         { *m = CancelInvoiceMsg{} }
 func (m *CancelInvoiceMsg) String() string { return proto.CompactTextString(m) }
 func (*CancelInvoiceMsg) ProtoMessage()    {}
 func (*CancelInvoiceMsg) Descriptor() ([]byte, []int) {
-	return fileDescriptor_invoices_b95f8bc86c534ce9, []int{0}
```

Comment:
A wrong version was used in a commit already in master.

Reply:
IIRC every time you compile it gets regenerated now as it seems to include a timestamp somewhere?

Reply:
Possible that it includes a timestamp, but only if any changes were actually made to the relevant protos. In this case the diff was attempting to fix an incorrect version used, but this is no longer needed since it's been addressed in master.